server/odc-service/src/main/java/com/oceanbase/odc/service/session/model/SqlExecuteResult.java

```diff
@@ -216,7 +216,8 @@ private boolean checkContainsExternalTable(@NonNull ConnectionSession connection
         DialectType dialectType = connectionSession.getDialectType();
         if (dialectType == DialectType.OB_MYSQL || dialectType == DialectType.OB_ORACLE) {
             String obVersion = ConnectionSessionUtil.getVersion(connectionSession);
-            if (VersionUtils.isGreaterThanOrEqualsTo(obVersion, MIN_OB_VERSION_FOR_EXTERNAL_TABLE)) {
+            if (VersionUtils.isGreaterThanOrEqualsTo(obVersion, MIN_OB_VERSION_FOR_EXTERNAL_TABLE)
```

Comment:
What's this change for?

Reply:
Used to fix npe about external table. Commit it together in this pr for convenience